js/main.js

```diff
@@ -1,2 +1,91 @@
 $(function () {
+  let random;
+
+  let stringBotScore = $("#computerScore").text()
```

Comment:
@cbrooks2019 looks like your attempting the read the score from the UI, I don't think that is necessary since your code is going to be responsible for determining the score